eosdac-material/src/pages/registercandidate.vue

```diff
@@ -66,6 +66,9 @@
 
           <q-btn size="md" v-if="getMemberRoles.candidate" class="animate-pop" :loading="loading" color="primary" @blur.native="userMsg=''" @click="unregisterAsCandidtate" :label="$t('regcandidate.unregister')">
             <q-spinner slot="loading" />
+          </q-btn>
+          <q-btn size="md"  class="animate-pop"  color="primary"  @click="unstake" label="test unstake">
```

Comment:
Is this ready for the master branch? If so, let's change the language and put it in a language file.

Reply:
hey luke, no I added that button to trigger an error (for testing the new contract errors). however the unstaking is working so it wouldn't take to long to make the unstaking production ready. I can do that tomorrow.